health-services/ExerciseSampleCompose/app/src/main/java/com/example/exercisesamplecompose/presentation/exercise/ExerciseScreen.kt

```diff
@@ -162,6 +164,11 @@ fun ExerciseScreen(
             }
         }
     }
+    //if we meet an exercise goal, show our exercise met dialog
+    if (uiState.exerciseState?.exerciseGoal == true) {
```

Comment:
I expect this will show again if the user switches away and switches back.

Probably not worth it for a sample, but should maybe comment that it's a simple version of what a product app would do.

See https://medium.com/androiddevelopers/viewmodel-one-off-event-antipatterns-16a1da869b95 if you want to understand more of the complexity.

Reply:
Done!